src/experimental/safe_html_transform/interfaces.py

```diff
@@ -2,6 +2,34 @@
 """Module where all interfaces, events and exceptions live."""
 
 from zope.publisher.interfaces.browser import IDefaultBrowserLayer
+from z3c.form import interfaces
+ 
+from zope import schema
+from zope.interface import Interface
+ 
+from zope.i18nmessageid import MessageFactory
+ 
+_ = MessageFactory('experimental.safe_html_transform')
+ 
+ 
+class IExperimenatalSafe_Html_TransformSettings(Interface):
+"""Global safe_html_transform settings. This describes records stored in the
+	configuration registry and obtainable via plone.registry.
+"""
+ 
+    Experimentalsafe_html_transform_key = schema.TextLine(title=_(u"Experimenatalsafe_html_transform (Wordpress) Key"),
```

Comment:
We should also create a module for controlpanel and then we can work
accordingly ?

Prakhar Joshi
DA-IICT,Gandhinagar

On Mon, Feb 9, 2015 at 10:31 PM, prakhar joshi <prakhar126@gmail.com> wrote:

> here its seems like the contropanel is a folder containing different
> files. Should we go like that ?
>
> Prakhar Joshi
> DA-IICT,Gandhinagar
>
> On Mon, Feb 9, 2015 at 10:28 PM, Timo Stollenwerk <
> notifications@github.com> wrote:
>
>> In src/experimental/safe_html_transform/interfaces.py
>> <https://github.com/collective/experimental.safe_html_transform/pull/1#discussion_r24344029>
>> :
>>
>> > +from z3c.form import interfaces
>> > +
>> > +from zope import schema
>> > +from zope.interface import Interface
>> > +
>> > +from zope.i18nmessageid import MessageFactory
>> > +
>> > +_ = MessageFactory('experimental.safe_html_transform')
>> > +
>> > +
>> > +class IExperimenatalSafe_Html_TransformSettings(Interface):
>> > +"""Global safe_html_transform settings. This describes records stored in the
>> > +	configuration registry and obtainable via plone.registry.
>> > +"""
>> > +
>> > +    Experimentalsafe_html_transform_key = schema.TextLine(title=_(u"Experimenatalsafe_html_transform (Wordpress) Key"),
>>
>> You should basically just copy the code from this pull request into our
>> new package: plone/Products.CMFPlone@ac98b0e
>> <https://github.com/plone/Products.CMFPlone/commit/ac98b0ea4ae27f4b74f4d3c5cf100b54f8acbfb7>
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/collective/experimental.safe_html_transform/pull/1/files#r24344029>
>> .
>>
>
>